models/metadata.go

```diff
@@ -0,0 +1,83 @@
+package models
+
+// Metadata represents information (metadata) relevant to a version
+type Metadata struct {
+	Alerts            *[]Alert             `json:"alerts,omitempty"`
```

Comment:
I assume these are all omitempty on the version and dataset documents? in future we may want more control around them, but as long as we're not inconsistently flexible i am happy with this 👍

Reply:
Correct